packages/opencensus-core/src/stats/view.ts

```diff
@@ -28,7 +28,7 @@ export class BaseView implements View {
   readonly measure: Measure;
   /**
    * A map of stringified tags representing columns labels or tag keys, concept
-   * similar to dimensions on multidimensional modeling, to AggregationData.
+   * similar to dimensions on multidimensional modeling, to AggregationMetadata.
```

Comment:
`AggregationMetadata` -> `AggregationData` in this file.

Reply:
Thanks, there were a few other occurrences of this mistake, fixed.